src/main/resources/config.yml

```diff
@@ -12,4 +12,9 @@ auto-reset: true
 any-death: false
 
 # How many lives a player should have before they are considered dead.
-lives: 1
\ No newline at end of file
+lives: 1
+
+# Set seed togle
+setSeed: false
+# Seed for the world to generate with.
+Seed: 0
```

Comment:
Inconsistent casing from all the other config values. use kebab-case like the other config options.